src/qibolab/pulses.py

```diff
@@ -468,41 +468,43 @@ class SNZ(PulseShape):
     """
     Sudden variant Net Zero.
     https://arxiv.org/abs/2008.07411
+    (Supplementary materials)
```

Comment:
maybe this would be enough

```suggestion
    (Supplementary materials: FIG. S1.)
```